### DISCUSSION TITLE
Is there an easy way to view the CSS output for troubleshooting?

### DISCUSSION THREAD

**Discussion by shaipetel** [microsoft/griffel#609] [Q&A]
Feels like I am shooting in the dark, unsure what CSS output my code produces and if a style isn't applied this makes it very hard to troubleshoot.

Looking in dev tools at the style classes, they all appear empty, can't view the CSS that was outputted.

Is there an easy way to view the complete CSS that is outputted by griffel to help troubleshooting issues?

**Comment by layershifter** [comment]
@shaipetel have you tried https://griffel.js.org/try-it-out/?
